core/amber/src/main/scala/edu/uci/ics/texera/web/model/websocket/request/WorkflowExecuteRequest.scala

```diff
@@ -6,7 +6,8 @@ import edu.uci.ics.texera.workflow.common.workflow.{BreakpointInfo, LogicalLink}
 case class WorkflowExecuteRequest(
     executionName: String,
     engineVersion: String,
-    logicalPlan: LogicalPlanPojo
+    logicalPlan: LogicalPlanPojo,
+    replayFromExecution: Option[Long]
```

Comment:
I think it is better to use a separated request. The semantic of `replayingFromExecution` indicates that we should use the old execution's information, including logicalPlan, executionName, and engineVersion. In another word, if I was to replay an execution, I should retrieve those information (logicalPlan, executionName, and engineVersion) from the previous execution information stored in db.

Reply:
Will do it later.